src/ordered-elements.js

```diff
@@ -1,5 +1,4 @@
 /* @flow */
-/* global Map */
 
 export default class OrderedElements {
```

Comment:
The branch coverage on this file falls to 75%, but I can't see any missed branches in the HTML report. I'm guessing it might not be getting the coverage right with the Babel transform happening here. Thoughts?